package.json

```diff
@@ -22,11 +22,17 @@
     "isomorphic"
   ],
   "author": "Reza Akhavan <jedireza@gmail.com> (http://reza.akhavan.me/)",
+  "contributors": [
+    "Tane Piper <piper.tane@gmail.com> (https://github.com/tanepiper)"
+  ],
```

Comment:
Happy to defer on this one,  it's mostly for the npm contributors graph and shows there but not that precious about it as you say it's on Github